src/MultipleComplaintImageFactory.js

```diff
@@ -203,6 +222,26 @@ export default function(Complaints, SelectImages) {
       let image = this.refs.image;
       image.width(width - imagePadding);
       image.height(width - imagePadding);
+
+      let { active } = this.state;
+      let complaints = this.state[active] ? this.state[active].complaints : [];
+      let srcWidth =
+        this.state.srcWidth !== undefined ? this.state.srcWidth : "";
```

Comment:
The number is being defaulted to a string.

Reply:
Done